cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,83 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if billingAccount == "" && !interactive.Enabled() {
+		billingAccount = aws.GetJumpAccount("local")
+		r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+		r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+	}
+
+	if interactive.Enabled() {
+		cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+		if err != nil {
+			r.Reporter.Errorf("%s", err)
+			os.Exit(1)
+		}
+
+		if len(cloudAccounts) > 0 {
+			billingAccount, err = interactive.GetOption(interactive.Input{
+				Question: "Billing Account",
+				Help:     cmd.Flags().Lookup("billing-account").Usage,
```

Comment:
the billing-account flag is currently hidden - we probably want to make it visible now.

Reply:
Enabled